protocols/datagram.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+// TODO:
+// Come up with a better name for this.
+package beavertail;
+
+message DatagramPush {
+  string busID                    = 10;
+  uint64 passengerCount           = 20;
+  double passengerCountConfidence = 30;
+  double latitude                 = 40;
+  double longitude                = 50;
+  int64  timestamp                = 60;  // Unix timestamp
```

Comment:
Should this be
```suggestion
  uint64 timestamp                = 60;  // Unix timestamp
```
?

Reply:
[Golang](https://golang.org/pkg/time/#Time.Unix) uses int64 for their time stamps so I think int64 should stay, might be different on the client side though

Reply:
Good point. Both `int64` and `uint64` are variable length anyway, so this change was more for hinting and clarity